api/views.py

```diff
@@ -1,7 +1,42 @@
 from django.shortcuts import render
 
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+
+from lyrics.models import Lyric
+from lyrics.serializers import LyricSerializer
+
+from django.http import HttpResponse, JsonResponse
+
 
 def test(request):
     template = ""
     context = {}
     return render(request=request, template_name=template, context=context)
+
+
+@api_view(["GET"])
+def Get_LyricsList(request): # new 

```

Comment:
this is not a good naming convention. functions should always be in small letters. You could just use `lyrics_list` that's fine